src/pages/StatusPage.vue

```diff
@@ -96,6 +96,15 @@
                     <prism-editor v-model="config.customCSS" class="css-editor" :highlight="highlighter" line-numbers></prism-editor>
                 </div>
 
+                <!-- Custom HTML -->
+                <div class="my-3">
+                    <div class="mb-1">{{ $t("Custom HTML") }}</div>
+                    <prism-editor v-model="config.customHtml" class="css-editor" :highlight="highlighter" line-numbers></prism-editor>
```

Comment:
I think the frontend should be `disabled` if said environment variable is not present and `enabled` when it is.

This would also enable baving a better helptext, communicating that adding stuff here is inherently kind of dangerous as said code is not vetted by security reviews.

Reply:
Since [vite also supports environment variables](https://vitejs.dev/guide/env-and-mode), I am not convinced that would be necessary.
Needs figuring out if this linked correctly though.

Reply:
Haven't tested this but I think these only work at build time.

Reply:
Also have not tested, but arent we using SSR for this feature?
=> the environment variables should be covered...